src/featureFlags.js

```diff
@@ -57,7 +58,10 @@ export class FeatureFlags extends events.EventEmitter {
       if (shouldIgnore) {
         return false;
       }
-      this.client.updateContext({ userId: this.userId });
+      this.client.updateContext({
+        userId: this.userId,
+        stage: this.network,
```

Comment:
💬 There is no actual stage in this project, right? Does it make sense to add?

Reply:
Hmmm, I don't think so as it would have to be manually inserted

I refactored `stage` to `network` though as it makes more sense: c7376a5834224194e61c83db9bfffd19bc4b045e